embrace-android-sdk/src/test/java/io/embrace/android/embracesdk/ndk/EmbraceNdkServiceTest.kt

```diff
@@ -541,11 +553,73 @@ internal class EmbraceNdkServiceTest {
     @Test
     fun `test initialization does not does not install signals and create directories if loadEmbraceNative is false`() {
         enableNdk(true)
-        every { sharedObjectLoader.loadEmbraceNative() } returns false
+        every { sharedObjectLoader.loadEmbraceNative() } returns true
+        initializeService()
+        verify(exactly = 0) { embraceNdkService["installSignals"]() }
```

Comment:
The previous implementation of this test didn't actually verify that no calls were made to the mock because they were invoked before the mock was even made, so it was always passing. irrespective what the object instantiation did. 

Changing this up to actually verified these aren't called.